qa/rolling-upgrade/src/test/java/org/opensearch/knn/bwc/StatsIT.java

```diff
@@ -14,10 +14,8 @@
 import java.util.List;
 import java.util.Map;
 
-import static org.opensearch.knn.plugin.stats.KNNStatsConfig.KNN_STATS;
-
 public class StatsIT extends AbstractRollingUpgradeTestCase {
-    private KNNStats knnStats = new KNNStats(KNN_STATS);
+    private KNNStats knnStats = new KNNStats();
```

Comment:
if you want to init this KNNStats the better way to do is using a function having with annotation @Before on top of it.

Reply:
Sure will update.